[IMP] amazon: migrate to SP-API (#1654)
odoo/documentation: content/applications/sales/sales/amazon_connector/update.rst

> Install the Amazon Connector Authentication Update

Reviewer: Shouldn't we change the title as well?

Install the Amazon Connector Authentication Update
==================================================

Starting July 2022, Amazon change it's API and the Amazon Connector needs to be updated to continue
to exchange information with Amazon. This also makes it easier for you to set up the Connector.

> This documentation will help you install the update and use the new authentication flow.

Reviewer 2: Let's delete this line. Normally we do not preface the documentation by saying what information will be included in each one.

> Note that if your Odoo database was first created after the update was released, the

Reviewer 2: Let's turn this into a Note to grab the reader's attention and separate this information from the rest of the instructions on the doc. Also, I'm editing the tense to keep it consistent throughout the sentence.
```suggestion
.. note::
If your Odoo database is created after the update was released, the
```

> update module is installed automatically. You can check if this module is already installed

Reviewer 2: Reworking this sentence for present tense and flow.
```suggestion
update module is installed automatically. To check if this module is already installed,
```

> by going to the :guilabel:`Apps` menu, removing the ``Apps`` search facet and search for ``amazon``.

Reviewer 2: Reworking this sentence (continued from the last comment) and fixing some Odoo terminology.
```suggestion
go to :guilabel:`Apps`, remove the ``Apps`` filter and search for ``Amazon``.
```

> =======================================

Reviewer: Same here :point_up:

.. warning::

>    You should never install new modules in your production database without testing them in a

Reviewer 2: Adding a word for clarity.
```suggestion
   You should never install new modules in your production database without first testing them in a
```

   duplicate or staging environment. For Odoo.com customers, a duplicate database can be created
   from the database management page. For Odoo.sh users, you should use a staging or duplicate

>    database. For on-premise users, you should use a staging environment--contact your integrating

Reviewer 2: Let's fix this punctuation either with a semicolon `;` or an em dash `---` as we discussed :)
```suggestion
   database. For on-premise users, you should use a staging environment---contact your integrating
```

>     you should use a staging environment - you should contact your integrating partner for

Reviewer 2: Editing wording and punctuation.
```suggestion
    you should use a staging environment--contact your integrating partner for
```

Author: I understand why we should take off `you should` here, but I'm not too sure about the replacement, is it something I'm not english-speakery enough to know about? :sweat_smile:  I'd have used `:` instead, I think. I put your suggestion anyway, but keep me updated so that I can change it if necessary (or learned an interesting english uses of the punctuation :smile: ). 

(BTW, since we are changing things in this paragraph, I fixed the indentation as well, so don't mind the huge diff, I only changed what you told me to ^^)

Author: Ho, nice, indeed, `---` did the trick :partying_face:

    more information regarding how to test a new module in your particular setup.

> The module should now be available in your :guilabel:`Apps` menu. Remove the ``Apps`` search facet
> and search for ``amazon``; the module :guilabel:`Amazon/Selling Partner API Patch` should be
> available for installation. If you cannot find the module after having updated the list of available

Reviewer 2: Fixing some Odoo terminology.
```suggestion
The module should now be available in your :guilabel:`Apps`. Remove the ``Apps`` filter
and search for ``Amazon``; the module :guilabel:`Amazon/Selling Partner API Patch` should be
available for installation. If you cannot find the module after having updated the list of available
```

modules, it means your Odoo source code is not up-to-date; refer to step one of this
page.

> Once the module is installed, you will need to link your account by clicking on a button which will
> redirect you to your Amazon Account. This process is detailed in the :doc:`setup <setup>` page.

Reviewer 2: Editing for brevity and instructional voice.
```suggestion
Once the module is installed, link your Amazon account by following the instructions on the :doc:`setup <setup>` page.
```
